code/lib/preview-api/src/modules/store/csf/portable-stories.test.ts

```diff
@@ -14,8 +21,57 @@ describe('composeStory', () => {
     },
   };
 
+  it('should call and compose loaders data', async () => {
+    const loadSpy = vi.fn();
+    const args = { story: 'story' };
+    const LoaderStory: Story = {
+      args,
+      render: (_args, { loaded }) => {
+        expect(loaded).toEqual({ foo: 'bar' });
+      },
+      loaders: [
+        async (context) => {
+          loadSpy();
+          expect(context.args).toEqual(args);
+          return {
+            foo: 'bar',
+          };
+        },
+      ],
```

Comment:
Let's put loaders first

```suggestion
      loaders: [
        async (context) => {
          loadSpy();
          expect(context.args).toEqual(args);
          return {
            foo: 'bar',
          };
        },
      ],
      render: (_args, { loaded }) => {
        expect(loaded).toEqual({ foo: 'bar' });
      },
```